### DISCUSSION TITLE
Instruction for using WSL2

### DISCUSSION THREAD

**Discussion by BatD2** [vscode-neovim/vscode-neovim#1014] [Q&A]
Can we please have detailed instructions of how to run the extension in WSL ? 
I think I followed all available informaiton, installed nvim 0.8, enabled wsl setting in the extenstion and added the linux path to nvim, but still getting error when loading it.

**Comment by Joamus** [comment]
I know this is necroing an old post, but I stumbled into a similar issue as well when setting up this plugin - from the sparse description from OP, I don't know if the issue is the same.

The problem I encountered was due to my Ubuntu distro not being set up as my "default distro" on WSL, because I had installed Docker prior to setting up my Ubuntu installation.

The solution I found was from some YouTube video, but its instructions were the same as this article's: https://winaero.com/set-default-wsl-linux-distro-windows-10/

For future reference, in case the article is no longer available:

Write "wsl --list --all" in your terminal, to get a list of Linux distros  you have installed

Afterwards, write "wsl --setdefault <distro name>" for the distro you are using.

It should now work :-)
